doc/iterations/2024.11/add_undo_redo_support.adoc

```diff
@@ -38,6 +38,34 @@ Pressing ctrl+y will redo the action that was previously undone.
 
 This should work regardless of the representation.
 
+=== Scenario (serialized representation)
+
+Serialized representation currently used in Sirius Web contains a layout part that is not contained in an EMF resource.
+
+On an unsynchronized representation the same scenarios as above should work.
+
+1. Several users are working on the same node 1
+- An user u1 resize a node from 10*10 to 100*100
+- An user u2 add sub nodes to the same node
+- If An user u1 press ctrl+z
+- *Expected*: the node will be not be resize to 10*10 if the sub nodes added by u2 do not fit inside, it will be resized to a default size.
+
+2. Several users are working on the same node 2
```

Comment:
I'm not sure scenarios 2 and 3 are realistically doable.
For example 2, it would require knowing that the action performed was : moving a node from 0,0 to 10,10 meaning for one action having the initial state and the resulting state.